numpy/ma/core.py

```diff
@@ -1430,7 +1427,7 @@ def getmask(a):
 get_mask = getmask
 
 
-def getmaskarray(arr):
+def getmaskarray(arr, nomask_as_readonly=False):
```

Comment:
Done in `make_mask_none`.

The real meaning in this function though is `require_to_be_writeable`, which I thought was better spelt `allow_readonly`.